src/app/money/import/mappings.ts

```diff
@@ -7,8 +7,8 @@ import { FormatMapping, FormatMappingBuilder } from "./format-mapping";
 // TODO Once we upgrade to TypeScript 3.4+, this can be rewritten to:
 // const ALL_FILE_FORMATS = ['foobar', ...] as const;
 // type FileFormat = typeof ALL_FILE_FORMATS;
-export const ALL_FILE_FORMATS = ['ksk_camt', 'ksk_creditcard', 'mlp', 'dkb', 'ubs', 'deutsche_bank', 'ing', 'n26'];
-export type ImportFileFormat = 'ksk_camt' | 'ksk_creditcard' | 'mlp' | 'dkb' | 'ubs' | 'deutsche_bank' | 'ing' | 'n26';
+export const ALL_FILE_FORMATS = ['ksk_camt', 'ksk_creditcard', 'mlp', 'dkb', 'ubs', 'deutsche_bank', 'ing', 'n26', 'genENG'];
+export type ImportFileFormat = 'ksk_camt' | 'ksk_creditcard' | 'mlp' | 'dkb' | 'ubs' | 'deutsche_bank' | 'ing' | 'n26' | 'genENG';
```

Comment:
Please follow camel_case naming of other types and name it `generic_en`. `en` is the ISO abbreviation of the language so it will be easier later to add other languages consistently.